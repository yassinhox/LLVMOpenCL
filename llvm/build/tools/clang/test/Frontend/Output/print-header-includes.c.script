/home/jazouani/llvm/build/Release+Asserts/bin/clang -cc1 -internal-isystem /home/jazouani/llvm/build/Release+Asserts/bin/../lib/clang/3.2/include -include Inputs/test3.h -E -H -o /home/jazouani/llvm/build/tools/clang/test/Frontend/Output/print-header-includes.c.tmp.out /home/jazouani/llvm/llvm-3.2/tools/clang/test/Frontend/print-header-includes.c 2> /home/jazouani/llvm/build/tools/clang/test/Frontend/Output/print-header-includes.c.tmp.err &&
FileCheck < /home/jazouani/llvm/build/tools/clang/test/Frontend/Output/print-header-includes.c.tmp.err /home/jazouani/llvm/llvm-3.2/tools/clang/test/Frontend/print-header-includes.c
