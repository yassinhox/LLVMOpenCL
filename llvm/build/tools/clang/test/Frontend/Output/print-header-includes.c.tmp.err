. /home/jazouani/llvm/llvm-3.2/tools/clang/test/Frontend/Inputs/test.h
.. /home/jazouani/llvm/llvm-3.2/tools/clang/test/Frontend/Inputs/test2.h
