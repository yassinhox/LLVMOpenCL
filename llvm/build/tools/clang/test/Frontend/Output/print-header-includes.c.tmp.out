# 1 "/home/jazouani/llvm/llvm-3.2/tools/clang/test/Frontend/print-header-includes.c"
# 1 "/home/jazouani/llvm/llvm-3.2/tools/clang/test/Frontend/print-header-includes.c" 1
# 1 "<built-in>" 1
# 1 "<built-in>" 3
# 150 "<built-in>" 3
# 1 "<command line>" 1
# 1 "/home/jazouani/llvm/llvm-3.2/tools/clang/test/Frontend/Inputs/test3.h" 1
int y;
# 2 "<command line>" 2
# 1 "<built-in>" 2
# 1 "/home/jazouani/llvm/llvm-3.2/tools/clang/test/Frontend/print-header-includes.c" 2







# 1 "/home/jazouani/llvm/llvm-3.2/tools/clang/test/Frontend/Inputs/test.h" 1
# 1 "/home/jazouani/llvm/llvm-3.2/tools/clang/test/Frontend/Inputs/test2.h" 1
int x;
# 2 "/home/jazouani/llvm/llvm-3.2/tools/clang/test/Frontend/Inputs/test.h" 2
# 9 "/home/jazouani/llvm/llvm-3.2/tools/clang/test/Frontend/print-header-includes.c" 2

